src/ExpenseTrackerGrupo2/Controllers/ExpenseController.cs

```diff
@@ -1,6 +1,138 @@
+using ExpenseTrackerGrupo2.Business.Services.Interfaces;
+using ExpenseTrackerGrupo2.Business.Services.Mappers.Requests.Expenses;
+
+using Microsoft.AspNetCore.Mvc;
+
 namespace ExpenseTrackerGrupo2.Controllers;
 
-public class ExpenseController
+[ApiController]
+[Route("api/expenses")]
```

Comment:
I suggest to change our path to api/v1/expenses

Reply:
Done